ext/standard/php_type.h

```diff
@@ -22,6 +22,7 @@ PHP_FUNCTION(floatval);
 PHP_FUNCTION(strval);
 PHP_FUNCTION(boolval);
 PHP_FUNCTION(gettype);
+PHP_FUNCTION(get_debug_type);
```

Comment:
Where is this function added to the function table?

Reply:
basic_functions.c, line 490

Reply:
Literally the first change - sorry I missed that :-)